src/Whoops/Util/SystemFacade.php

```diff
@@ -20,23 +20,19 @@ public function startOutputBuffering()
 
     /**
      * @param callable $handler
-     * @param int      $types
+     * @param int      $types By default E_ALL | E_STRICT
      *
-     * @return callable|null
+     * @return string|null
      */
-    public function setErrorHandler(callable $handler, $types = 'use-php-defaults')
+    public function setErrorHandler(callable $handler, $types = null)
     {
-        // Workaround for PHP 5.5
```

Comment:
If I blame the removed code, I find that this workaround was put here on a very explicit purpose in e18d217. Whilst the proposed changes do not fail anymore, the note about it being a workaround is gone – it is easy to expect the next developer "simplifying" the code further and reverting it to the pre-workaround version.

Instead, I would rather keep the very explicit conditional.